libyul/Utilities.cpp

```diff
@@ -42,6 +42,8 @@ using boost::is_any_of;
 
 string yul::reindent(string const& _code)
 {
+	int constexpr IndentationWidth = 4;
```

Comment:
```suggestion
	int constexpr indentationWidth = 4;
```

Reply:
for consts, I prefer ALL_CAPS or at least CamelCase. I knew this was going to be raised when the `git push` was already done. :-)

Reply:
all caps is ugly and upper-case camel case is reserved for types :)